content/app/events/event.js

```diff
@@ -0,0 +1,35 @@
+(function (window) {
+
+// imports
+
+/**
+ * Event constructor.
+ * @param {String} type
+ * @param {Object} data
+ */
+var Event = function (type, data) {
+	this.type = type;
+	this.data = data;
+	this.target = null;
+};
+
+/**
+ * Close event type.
+ * @type {String}
+ */
+Event.CLOSE = 'close';
```

Comment:
Maybe it would be better to have more specific constant name, like Event.TYPE_CLOSE or smilar. Because I would expect that Event.CLOSE will return predefined object for close event.